src/instruments/src/ND/pages/ArcMode.tsx

```diff
@@ -203,6 +201,10 @@ const ArcModeOverlayDefs = memo(() => (
         <clipPath id="arc-mode-overlay-clip-1">
             <path d="m 0 519 l 384 145 l 384 -86 v -580 h -768 z" />
         </clipPath>
+        {/* inverted map overlays for terrain map in WASM module  */}
+        <path name="arc-mode-bottom-left-map-area" d="M0,625 L122,625 L174,683 L174,768 L0,768 L0,625" className="nd-inverted-map-area" fill="rgb(4,4,5)" />
+        <path name="arc-mode-bottom-right-map-area" d="M768,562 L648,562 L591,625 L591,768 L768,768 L768,562" className="nd-inverted-map-area" fill="rgb(4,4,5)" />
+        <path name="arc-mode-top-map-area" d="M0,0 L0,312 a492,492 0 0 1 768,0 L768,0 L0,0" className="nd-inverted-map-area" fill="rgb(4,4,5)" />
```

Comment:
We should use the CSS definitions for the fill/background colour rather than hardcoding the colour here. These will change again soon it seems.

Reply:
fixed